packages/api/test/functional/user.spec.js

```diff
@@ -238,6 +242,20 @@ test('PUT /users/:id endpoint fails when trying update with same user email', as
 	);
 });
 
+test('PUT /users/:id endpoint admin user to try update status', async ({ client }) => {
+	const loggeduser = await User.create(adminUser);
+
+	const userInst = await User.create(user);
+
+	const response = await client
+		.put(`/users/${userInst.id}`)
+		.send({ ...userInst, status: 'verified' })
+		.loginVia(loggeduser, 'jwt')
+		.end();
+	response.assertStatus(200);
+	response.assertJSONSubset(userInst.toJSON());
```

Comment:
Esse teste é pra testar oq? se `userInst` for criado com status diferente de verified ele não vai ser alterado automaticamente depois que vc faz a requisição. Então acho que esse teste está passando quando não deveria. Acho interessante, pra ser bem explícito verificar se de fato o status do usuário foi alterado para `verified`

Reply:
Faz sentido. Deixa explícito o status inicial e depois checar se mudou.